Content.Shared/Weapons/Ranged/Systems/SharedGunSystem.cs

```diff
@@ -389,6 +390,12 @@ protected void SetCartridgeSpent(EntityUid uid, CartridgeAmmoComponent cartridge
         xform.LocalRotation = Random.NextAngle();
         xform.Coordinates = coordinates;
 
+        // decides direction the casing ejects and only when not cycling
+        if (angle != 0f)
```

Comment:
just make the angle nullable, in that case.

Reply:
When I try to make it nullable I get this:
![image](https://github.com/space-wizards/space-station-14/assets/137322659/52e6bcd8-793c-4b73-8a7f-549dcf70b2b3)

Reply:
Use null as default.

Reply:
It throws the same errors if I make it null

![image](https://github.com/space-wizards/space-station-14/assets/137322659/4e26bb0f-316f-4fdc-8809-cb94c7208847)
![image](https://github.com/space-wizards/space-station-14/assets/137322659/ee82096a-12e3-4b3f-9409-e5598039655c)

Reply:
I've fixed the null-ness problems.